Remotion/Web/Core/ExecutionEngine/WxeHandler.cs

```diff
@@ -162,18 +162,17 @@ protected void CheckTimeoutConfiguration (HttpContext context)
       if (! IsSessionManagementEnabled)
         return;
 
-      int functionTimeout = WebConfiguration.Current.ExecutionEngine.FunctionTimeout;
+      int functionTimeout = SafeServiceLocator.Current.GetInstance<IWxeLifetimeManagementSettings>().FunctionTimeout;
       if (functionTimeout > context.Session.Timeout)
         throw new WxeException("The FunctionTimeout setting in the configuration must not be greater than the session timeout.");
-      int refreshInterval = WebConfiguration.Current.ExecutionEngine.RefreshInterval;
+      int refreshInterval = SafeServiceLocator.Current.GetInstance<IWxeLifetimeManagementSettings>().RefreshInterval;
```

Comment:
Previous code didn't do it but we can move the resolved service into a variable to not look up twice